client/src/com/mirth/connect/client/ui/LoginPanel.java

```diff
@@ -575,7 +575,8 @@ private boolean handleSuccess(LoginStatus loginStatus) throws ClientException {
                     userPreferences = client.getUserPreferences(currentUser.getId(), preferenceNames);
 
                     // Display registration dialog if it's the user's first time logging in
-                    String firstlogin = userPreferences.getProperty("firstlogin");
+                    // TODO Re-implement a statistics service. See https://github.com/OpenIntegrationEngine/engine/issues/25
+                    String firstlogin = "false"; // userPreferences.getProperty("firstlogin");
```

Comment:
issue: we can't skip first login

users should be required to change the admin password on first login